docs/architecture/daemon-split.md

```
@@ -148,53 +166,48 @@ services:
-```yaml
-extra_hosts:
-  - "host.docker.internal:host-gateway"
-environment:
-  DAEMON_URL: http://host.docker.internal:9999
-```
-Requires Docker 20.10+
+The standalone `bosun webhook` receiver connects to the daemon via Unix socket or TCP API.
 
-**Option 2: Bridge Gateway IP**
+**Unix socket (recommended):**
 ```yaml
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)